nubi/src/components/header/menu/styles.module.scss

```
@@ -0,0 +1,30 @@
+@import '../../../constants/colors.module.scss';
+
+.menu {
+  background-color: $main-blue;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 35%;
+  min-width: 60vh;
+  padding: 3vh;
+  padding-top: 10vh;
+  position: absolute;
+  text-align: left;
+  top: 0;
+  right: 0;
+}
+
+.menuText {
+  color: white;
+  font-family: 'Rubik';
+  font-style: italic;
+  font-size: xx-large;
+  padding: 1vh;
+  width: fit-content;
+  text-decoration: none;
+
+  &:hover{
```

Cuidado con los espacios.
```suggestion
  &:hover {
```